mod-users/src/main/resources/domain/mod-users/features/util/initData.feature

```diff
@@ -0,0 +1,25 @@
+Feature: init data for mod-users
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
+    * def patronId = call uuid1
+    * def patronName = call random_string
+    * def barCode = call random_numbers
+
+  @PostPatronGroupAndUser
+  Scenario: create PatronGroup & User
+    * def createPatronGroupRequest = read('samples/PatronGroup/create-patronGroup-request.json')
+    Given path 'groups'
+    And request createPatronGroupRequest
+    When method POST
+    Then status 201
+    * def patronGroupId = response.id
+
+    * def createUserRequest = read('samples/User/create-user-request.json')
+    * createUserRequest.active = karate.get('status')
```

Comment:
What does `karate.get` do? Is this how parameters are expressed in these executable chunks of code?

Reply:
Karate.get() is used to get  value safe mode, but please ignore that i added for something to test and forgot to remove , i will make correction in next commit. thankyou.